### PR TITLE
Adding checkmarks to DKG status output

### PR DESCRIPTION
**Pull request by AnomalRoil**
Just to make it prettier and clearer who has accepted and who hasn't accepted yet.

### CODE REVIEW

File: internal/drand-cli/dkg_cli.go [hunk 1]
```diff
@@ -559,8 +559,18 @@ func convert(entry *drand.DKGEntry) printModel {
...
-			b.WriteString(fmt.Sprintf("\n\t%s,", a.Address))
+			if checks && util.Contains(entry.Remaining, a) {
+				checkmark = " ☐ "
+				if util.Contains(entry.Acceptors, a) {
+					checkmark = " ☑ "
+				} else if util.Contains(entry.Rejectors, a) {
+					checkmark = " ☒ "
+				}
+			}
+			b.WriteString(fmt.Sprintf("\n\t%s%s,", a.Address, checkmark))
```
Comment (CluEleSsUK) [inline internal/drand-cli/dkg_cli.go:573]: might be best to put the checkmark first or inconsistent address length may make it hard to read

File: internal/drand-cli/dkg_cli.go [hunk 2]
```diff
@@ -545,11 +545,11 @@ type printModel struct {
 	FinalGroup  string
 }
 
-func convert(entry *drand.DKGEntry) printModel {
+func convert(entry *drand.DKGEntry, checks bool) printModel {
```
Comment (CluEleSsUK) [inline internal/drand-cli/dkg_cli.go:548]: bool params are ugly - a separate function for rendering checkmarks is clearer imo

**Comment by AnomalRoil** [comment]
This now prints:
``` 

+-------------+------------------------------+-----------------------------+
| FIELD       | CURRENT                      | FINISHED                    |
+-------------+------------------------------+-----------------------------+
| Status      | Complete                     | Complete                    |
| Epoch       | 2                            | 2                           |
| BeaconID    | default                      | default                     |
| Threshold   | 5                            | 5                           |
| Timeout     | 2024-06-13T10:24:16Z         | 2024-06-13T10:24:16Z        |
| GenesisTime | 2024-06-12T10:23:46Z         | 2024-06-12T10:23:46Z        |
| GenesisSeed | <snip>                       | <snip>                      |
| Leader      | 127.0.0.1:45777              | 127.0.0.1:45777             |
| Joining     | [                            | [                           |
|             |     127.0.0.1:42427,         |      ☑ 127.0.0.1:42427,     |
|             |     127.0.0.1:46847,         |      ☒ 127.0.0.1:46847,     |
|             |     127.0.0.1:34459,         |      ☑ 127.0.0.1:34459,     |
|             | ]                            | ]                           |
| Remaining   | [                            | [                           |
|             |      ☑ 127.0.0.1:45777,      |      ☑ 127.0.0.1:45777,     |
|             |      ☑ 127.0.0.1:34565,      |      ☑ 127.0.0.1:34565,     |
|             |      ☑ 127.0.0.1:43127,      |      ☑ 127.0.0.1:43127,     |
|             |      ☑ 127.0.0.1:41179,      |      ☑ 127.0.0.1:41179,     |
|             |      ☑ 127.0.0.1:46745,      |      ☑ 127.0.0.1:46745,     |
|             | ]                            | ]                           |
| Leaving     | [    127.0.0.1:46239    ]    | [    127.0.0.1:46239    ]   |
| Accepted    | [                            | [                           |
|             |     127.0.0.1:34565,         |     127.0.0.1:34565,        |
|             |     127.0.0.1:43127,         |     127.0.0.1:43127,        |
|             |     127.0.0.1:41179,         |     127.0.0.1:41179,        |
|             |     127.0.0.1:46745,         |     127.0.0.1:46745,        |
|             | ]                            | ]                           |
| Rejected    | []                           | []                          |
| FinalGroup  |                              | [                           |
|             |                              |     127.0.0.1:43127,        |
|             |                              |     127.0.0.1:42427,        |
|             |                              |     127.0.0.1:34565,        |
|             |                              |     127.0.0.1:41179,        |
|             |                              |     127.0.0.1:45777,        |
|             |                              |     127.0.0.1:46745,        |
|             |                              |     127.0.0.1:34459,        |
|             |                              | ]                           |
+-------------+------------------------------+-----------------------------+

```

**Comment by CluEleSsUK** [comment]
Hmm looking at the printed output we could maybe just get rid of the `Accepted`/`Rejected` rows now that we have cool checkmarks

File: internal/core/util_test.go [hunk 3]
```diff
@@ -401,7 +401,7 @@ func (d *DrandTestScenario) RunDKG(t *testing.T) (*key.Group, error) {
 	d.AdvanceMockClock(d.t, d.nodes[0].daemon.opts.dkgKickoffGracePeriod)
 
 	t.Log("[RunDKG] WaitForDKG on leader")
-	groupFile, err := d.WaitForDKG(t, leader, 1, 100)
+	groupFile, err := d.WaitForDKG(t, leader, 1, 150)
```
Comment (CluEleSsUK) [inline internal/core/util_test.go:404]: attempted flake combat? :p

**Comment by CluEleSsUK** [comment]
Ahh the checkmarks in the `Finished` state represent whether they made it to the final group too, that's a nice touch

**Pull request merged**